src/lib/storage/fake_device_factory.rb

```diff
@@ -281,9 +287,9 @@ def create_partition(parent, args)
 
         # partitions are created in order, so first suitable slot should be fine
         # note: skip areas we marked as empty
-        slot = slots.find { |slot|
-          slot.possible?(type) && !@free_regions.member?(slot.region.start)
-        }
+        slot = slots.find do |s|
```

Comment:
I liked the old version better. Was that something Rubocop wanted to be changed? Or was it a line length limitation?

IHMO that only shows how absurd many of the Rubocop settings are.